src/main/java/edu/iu/dlib/amppd/model/MgmModeIo.java

```diff
@@ -16,10 +17,10 @@
  * @author yingfeng
  *
  */
-@Entity
+@MappedSuperclass
 @Data
-@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
-@DiscriminatorColumn(name = "type")
+//@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+//@DiscriminatorColumn(name = "type")
 public abstract class MgmModeIo {
 
     @Id
```

Comment:
Can we revert this change on MgmModeIo? I think we do want to have both input/output in one table for now. If we feel there is a need to split later we can change then.

Reply:
Pushed the fixes